processing/src/main/java/org/apache/druid/segment/UnnestStorageAdapter.java

```diff
@@ -120,27 +117,27 @@ public Sequence<Cursor> makeCursors(
                   retVal,
                   retVal.getColumnSelectorFactory(),
                   unnestColumn,
-                  outputColumnName,
-                  allowSet
+                  outputColumnName
               );
             } else {
               retVal = new UnnestColumnValueSelectorCursor(
                   retVal,
                   retVal.getColumnSelectorFactory(),
                   unnestColumn,
-                  outputColumnName,
-                  allowSet
+                  outputColumnName
               );
             }
           } else {
             retVal = new UnnestColumnValueSelectorCursor(
                 retVal,
                 retVal.getColumnSelectorFactory(),
                 unnestColumn,
-                outputColumnName,
-                allowSet
+                outputColumnName
             );
           }
+          // This is needed at this moment for nested queries
+          // Future developer would want to move the virtual columns
```

Comment:
Why would a future developer want to do this? (Not a rhetorical question: I really don't know.) Please add some rationale to the comment so people know what you have in mind.

Reply:
My bad removed the stale comments and updated the docs